enigma-cli/build.gradle

```diff
@@ -1,20 +1,20 @@
 plugins {
-    id 'application'
-    id 'com.github.johnrengelman.shadow' version '7.0.0'
+	id 'application'
+	id 'com.github.johnrengelman.shadow' version '7.0.0'
 }
 
 dependencies {
-    implementation project(':enigma')
+	implementation project(':enigma')
 }
 
 mainClassName = 'cuchaz.enigma.command.Main'
 
 jar.manifest.attributes 'Main-Class': mainClassName
 
 publishing {
-    publications {
-        shadow(MavenPublication) { publication ->
-            project.shadow.component publication
-        }
-    }
+	publications {
+		shadow(MavenPublication) {publication ->
```

Comment:
Is it intentional that the spaces are removed after braces in the gradle scripts? (For example the lambda param here and the `url` method call in the fabric maven for the root build.gradle)

Reply:
No, I think I fxed all the spots. 👍